llvm/lib/Target/AMDGPU/SIDefines.h

```diff
@@ -87,9 +87,9 @@ enum : uint64_t {
   FLAT = 1 << 24,
   DS = 1 << 25,
 
-  // Pseudo instruction formats.
-  VGPRSpill = 1 << 26,
-  SGPRSpill = 1 << 27,
+  // Combined SGPR/VGPR Spill bit
+  // logic to seperate them out is done in isSGPRSpill and isVGPRSpill
```

Comment:
Capitalise "Logic" or write it as a complete sentence (per coding standards). Typo "separate".